app/models/engage_and_learn_api_token.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class EngageAndLearnApiToken < ApiToken
+  attribute :private_api_access, default: true
```

Comment:
Can we also add a migration to change existing E&L tokens to have private access? I just added a token to staging that should last for a while, and if things go smoothly I wanna add another one to production soon.

Reply:
Sure. Therefore I'm assuming we're happy doing migrations that change data.

Reply:
I sure am

Reply:
Definitely wrote some on E&L, pretty sure I saw some on R&P, in particular one that didn't work so well on the sandbox environment recently 😂 
But in principle, yes, I think they are too useful a tool to avoid :)

Reply:
done